eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -100,8 +100,10 @@ parameters:
 
 stages:
 - ${{ each cloud in parameters.CloudConfig }}:
+  # TODO: re-enable tests-weekly allow filter once sovereign cloud live tests are stable
```

Comment:
Do you have an issue tracking that? Might be a good idea to include a link to one.

Reply:
Updated comment to link to https://github.com/Azure/azure-sdk/issues/2074